pkg/worker/worker.go

```diff
@@ -262,30 +262,39 @@ func (s *Worker) shouldShutDown(lastContainerRequest time.Time) bool {
 
 // Spawn a single container and stream output to stdout/stderr
 func (s *Worker) RunContainer(request *types.ContainerRequest) error {
-	containerID := request.ContainerId
+	containerId := request.ContainerId
 
 	bundlePath := filepath.Join(s.imageMountPath, request.ImageId)
 
+	f, err := NewLogger(request)
+	if err != nil {
+		return err
+	}
+	defer f.logFile.Close()
+
 	// Pull image
-	log.Printf("<%s> - lazy-pulling image: %s\n", containerID, request.ImageId)
-	err := s.imageClient.PullLazy(request)
+	f.Log("loading image: %s", request.ImageId)
```

Comment:
I don't know that we need these logs sent to the user -- imo we only need the ones generated in the image client (that were tied directly to blobfs

Reply:
I bumped the logging down to be only in the image client.